docs/metrics.md

```diff
@@ -126,8 +126,8 @@ scp.timeout.nominate                     | meter     | timeouts in nomination
 scp.timeout.prepare                      | meter     | timeouts in ballot protocol
 scp.timing.nominated                     | timer     | time spent in nomination
 scp.timing.externalized                  | timer     | time spent in ballot protocol
-scp.timing.externalize-lag               | timer     | delay between first externalize message and local node externalizing
-scp.timing.externalize-delay             | timer     | delay between local node externalizing and later externalize messages
+scp.timing.self-to-first-externalize-lag | timer     | delay between first externalize message and local node externalizing
+scp.timing.rest-to-self-externalize-lag  | timer     | delay between local node externalizing and later externalize messages from other nodes
```

Comment:
this one also is not super clear. Maybe `self-to-other-externalize-lag` *or* `other-from-self-externalize-lag`

Reply:
same than earlier "to" reads more naturally I think